src/openrct2/ride/Vehicle.h

```diff
@@ -379,20 +379,24 @@ enum : uint32_t
     VEHICLE_ENTRY_FLAG_NO_UPSTOP_WHEELS = 1 << 1,
     VEHICLE_ENTRY_FLAG_NO_UPSTOP_BOBSLEIGH = 1 << 2,
     VEHICLE_ENTRY_FLAG_MINI_GOLF = 1 << 3,
-    VEHICLE_ENTRY_FLAG_4 = 1 << 4,
-    VEHICLE_ENTRY_FLAG_5 = 1 << 5,
+    VEHICLE_ENTRY_FLAG_REVERSER_BOGIE = 1 << 4,
+    VEHICLE_ENTRY_FLAG_REVERSER_PASSENGER_CAR = 1 << 5,
     VEHICLE_ENTRY_FLAG_HAS_INVERTED_SPRITE_SET = 1 << 6, // Set on vehicles that support running inverted for extended periods
                                                          // of time, i.e. the Flying, Lay-down and Multi-dimension RCs.
     VEHICLE_ENTRY_FLAG_DODGEM_INUSE_LIGHTS = 1
         << 7, // When set the vehicle has an additional frame for when in use. Used only by dodgems.
     VEHICLE_ENTRY_FLAG_ALLOW_DOORS_DEPRECATED = 1 << 8, // Not used any more - every vehicle will now work with doors
     VEHICLE_ENTRY_FLAG_ENABLE_ADDITIONAL_COLOUR_2 = 1 << 9,
-    VEHICLE_ENTRY_FLAG_10 = 1 << 10,
-    VEHICLE_ENTRY_FLAG_11 = 1 << 11,
+    VEHICLE_ENTRY_FLAG_RECALCULATE_SPRITE_BOUNDS = 1 << 10, // Only used during loading of the objects
+    VEHICLE_ENTRY_FLAG_HALF_ROTATION_SPRITE_COUNT = 1
```

Comment:
This doesn't make it clear that it only effects flat sprites. I would rather it was called 16_FLAT_ROTATION_FRAMES

Reply:
> This doesn't make it clear that it only effects flat sprites.

Yes, but then, I think it should affect all places where it currently expects 32 rotations. Still though, using the number 16 explicitly is a good idea.

I thus propose `VEHICLE_ENTRY_FLAG_16_ROTATION_FRAMES`.

Reply:
Yes but it's specifically for flat sprites. In locomotion CS had separate fields for how many flat rotation sprites and how many sloped rotation sprites. This allows for a smooth rotation in the vehicle preview page whilst also not needing to generate all the sloped rotation frames.

Reply:
> Yes but it's specifically for flat sprites

One can argue either way, since it is only set on vehicles that only have flat sprites to begin with.

Reply:
I guess the proper question to ask is what do object makers want? Would they want the ability to specify how many rotation sprites there are for flat and specify a different amount for sloped?

Reply:
Ideally this flag would be replaced in the future by something more alike to Locomotion.

My thoughts with the name of flag and `VEHICLE_SPRITE_FLAG_ROTATION_COUNT_DIVIDE_BY_EIGHT` is that it seems that they can be used together to only use 2 rotation sprites, instead of 4 or 16. 

But if that is something you guys don't wanna support (no vanilla rides use them both together), then my proposal would be to change them to `VEHICLE_ENTRY_FLAG_USE_16_ROTATION_FRAMES` and `VEHICLE_SPRITE_FLAG_USE_4_ROTATION_FRAMES` (similar to Gymnasiasts proposal but including the word 'use' as well for easier readability).

Reply:
Hm, I prefer keeping to the slightly shorter one.

What do you think @duncanspumpkin ?

Reply:
Okay we are going to go with the suggestion of not including FLAT in the name. I dont mind too much about USE_ i guess it makes sense. We will modify the codebase so that this flag effects sloped sprites as well but in a seperate PR. Yes we do not support both flags being set.

Reply:
Alright, I've updated them accordingly. :)